tasks/rule_generator/src/main.rs

```diff
@@ -1,10 +1,11 @@
 use std::{borrow::Cow, fs::File, io::Write, path::Path, process::Command, rc::Rc};
 
 use convert_case::{Case, Casing};
+use lazy_static::lazy_static;
 use oxc_allocator::Allocator;
 use oxc_ast::{
     ast::{Argument, Expression, ObjectProperty, PropertyKey, PropertyValue},
-    AstKind, SourceType,
+    AstKind, GetSpan, SourceType,
```

Comment:
unused? @Boshen can we setup CI to lint/format against Rust files in `/tasks` as well? I think it would catch a few things here

Reply:
This is actually used to get the span of the js object value:
`let span = prop.value.span();`